ui/app/src/components/LauncherSiteCard/LauncherSiteCard.tsx

```diff
@@ -75,22 +79,29 @@ function LauncherSiteCard(props: LauncherSiteCardProps) {
     <>
       <Box
         // @ts-ignore
-        button
+        button={isSiteReady}
         selected={selected}
         boxShadow={1}
         component={ListItem}
-        onClick={() => onCardClick(value)}
+        onClick={() => isSiteReady && onCardClick(value)}
         className={cx(classes.card, props.classes?.root)}
+        sx={{ position: 'relative' }}
         title={title}
       >
-        <ListItemText primary={title} primaryTypographyProps={{ className: classes.siteName, noWrap: true }} />
-        {hasOptions && (
-          <ListItemSecondaryAction>
+        <ListItemText
+          primary={title}
+          primaryTypographyProps={{ className: classes.siteName, noWrap: true }}
+          sx={{ paddingRight: isSiteReady ? 0 : '35px' }}
+        />
+
+        <ListItemSecondaryAction>
+          {hasOptions && (
             <IconButton aria-label="settings" onClick={handleOptions} size="large">
               <MoreVertIcon />
             </IconButton>
-          </ListItemSecondaryAction>
-        )}
+          )}
+        </ListItemSecondaryAction>
+        {!isSiteReady && <SiteStatusIndicator state={state} sx={{ position: 'absolute', top: 22, right: 15 }} />}
```

Comment:
Let's replace the IconButton with the indicator when the site isn't ready, put the indicator in the secondary action slot and not use absolute position.
```tsx
{!isSiteReady||hasOptions && (
  <ListItemSecondaryAction>
    {isSiteReady ? (
      <IconButton/>
    ) : (
      <Indicator/>
    )}
```